src/widget-sources/types.ts

```diff
@@ -33,7 +33,8 @@ export interface BaseRequestOptions {
 /** Options for {@link WidgetRemoteSource#getCategories}. */
 export interface CategoryRequestOptions extends BaseRequestOptions {
   column: string;
-  operation?: 'count' | 'avg' | 'min' | 'max' | 'sum';
+  operation?: 'count' | 'avg' | 'min' | 'max' | 'sum' | 'custom';
```

Comment:
Optionally, could use `AggregationType` here since it now includes all values from the type union.

Also, do you mind including a `/** Remote only. */` comment similar to the one on `joinOperation`? I know users probably won't see it but this does help when I need to remember which operations we support in remote vs. tileset widgets. 🙏🏻